kubernetes/charts/weblogic-operator/templates/_operator-dep.tpl

```diff
@@ -165,3 +167,170 @@ spec:
             path: logstash.conf
       {{- end }}
 {{- end }}
+---
+  {{ $chartVersion := .Chart.Version }}
+  {{ $webhookExists := include "utils.verifyExistingWebhookDeployment" (list $chartVersion) | trim }}
+  {{- if or (ne $webhookExists "true") (.recreateWebhook) }}
+    # webhook does not exist or chart version is newer, create a new webhook
+    apiVersion: "v1"
+    kind: "ConfigMap"
+    metadata:
+      labels:
+        weblogic.webhookName: {{ .Release.Namespace | quote }}
+      name: "weblogic-webhook-cm"
+      namespace: {{ .Release.Namespace | quote }}
+    data:
+      serviceaccount: {{ .serviceAccount | quote }}
```

Comment:
I'm assuming that this is the ConfigMap where the webhook will store its certificate and keys, correct? If so, consider preserving the values from any earlier version of the map, if present, just like the operator does for its internal certificate:

```
  {{- $configmap := (lookup "v1" "ConfigMap" .Release.Namespace "weblogic-operator-cm") }}
  {{- if (and $configmap $configmap.data) }}
  {{- $internalOperatorCert := index $configmap.data "internalOperatorCert" }}
  {{- if $internalOperatorCert }}
  internalOperatorCert: {{ $internalOperatorCert }}
  {{- end }}
  {{- end }}
```

Reply:
Ignore this... You already used the suggestion in the webhook's secret.